org.eclipse.winery.frontends/app/tosca-management/src/app/instance/sharedComponents/files/fileManager/fileManager.component.ts

```diff
@@ -67,7 +66,11 @@ export class FileManagerComponent implements OnChanges {
     determineBaseDir() {
         const keys = Array.from(this.pathToElementsMap.keys());
         keys.sort((a, b) => a.length > b.length ? 1 : -1);
-        this.baseDir = keys[0];
+        if (keys[0] !== undefined) {
```

Comment:
In TypeScript, you can check for undefined by only doing this:

```ts
if (!keys[0]) {
   // keys[0] is undefined
} else {
    // keys[0] is set
}
```

Or even simpler:

```ts
this.baseDir = !!keys[0] ? keys[0] : ''
```